sdk/python/testvectors/__main__.py

```diff
@@ -1,28 +1,31 @@
+import copy
 import importlib
-import inspect
 import os
 from binascii import hexlify
 from pathlib import Path
 
+import sha3
 import yaml
 
+from symbolchain import nc, sc
+from symbolchain.ByteArray import ByteArray
 from symbolchain.facade.NemFacade import NemFacade
 from symbolchain.facade.SymbolFacade import SymbolFacade
 
-
-def read_test_vectors_file(filepath):
-	with open(filepath, 'rt', encoding='utf8') as input_file:
-		return yaml.safe_load(input_file)
+from .utils import calculate_transactions_hash
 
 
 def get_modules(network_name):
 	module_names = {
-		'nem': ['account_key_link', 'cosignature', 'mosaic_definition', 'mosaic_supply_change', 'multisig',
-			'multisig_account_modification', 'namespace_registration', 'transfer'],
-		'symbol': ['account_address_restriction', 'account_metadata', 'account_mosaic_restriction', 'account_operation_restriction',
-			'address_alias', 'aggregate_complete', 'aggregate_mosaic_supply_change', 'hash_lock', 'key_link', 'mosaic_address_restriction',
-			'mosaic_alias', 'mosaic_definition', 'mosaic_global_restriction', 'mosaic_metadata', 'mosaic_supply_change',
-			'multisig_account_modification', 'namespace_metadata', 'namespace_registration', 'secret_lock', 'secret_proof','transfer']
+		'nem': [
+			'account_key_link', 'cosignature', 'mosaic_definition', 'mosaic_supply_change', 'multisig',
+			'multisig_account_modification_v1', 'multisig_account_modification', 'namespace_registration', 'transfer_v1', 'transfer'],
+		'symbol': [
+			'account_address_restriction', 'account_key_link', 'account_metadata', 'account_mosaic_restriction',
+			'account_operation_restriction', 'address_alias', 'aggregate_bonded', 'aggregate_complete', 'hash_lock',
+			'mosaic_address_restriction', 'mosaic_alias', 'mosaic_definition', 'mosaic_global_restriction', 'mosaic_metadata',
+			'mosaic_supply_change', 'multisig_account_modification', 'namespace_metadata', 'namespace_registration', 'node_key_link',
+			'secret_lock', 'secret_proof', 'transfer', 'voting_key_link', 'vrf_key_link']
 	}
```

Comment:
any reason not to autodetect ?

Reply:
I thought you wanted explicit list?

Reply:
i don't remember that 😅 i think autodetecting should be fine